drake/examples/Cars/prius/prius.sdf

```diff
@@ -514,78 +461,80 @@
         <use_parent_model_frame>1</use_parent_model_frame>
       </axis>
     </joint>
-    
-    <joint name='rear_left_wheel_joint' type='revolute'>
-      <parent>rear_axle</parent>
-      <child>left_wheel_rear</child>
-      <pose>0 0 0 0 0 0</pose>
-      <axis>
-        <xyz>0 1 0</xyz>
-        <use_parent_model_frame>1</use_parent_model_frame>
-      </axis>
-    </joint>
-    
-    <joint name='rear_right_wheel_joint' type='revolute'>
-      <parent>rear_axle</parent>
-      <child>right_wheel_rear</child>
-      <pose>0 0 0 0 0 0</pose>
+    <joint name='axle_tie_rod_arm' type='revolute'>
+      <parent>front_axle</parent>
+      <child>right_tie_rod_arm</child>
+      <pose>0.17246968 -0.08623384 0 0 0 0</pose>
       <axis>
-        <xyz>0 1 0</xyz>
-        <use_parent_model_frame>1</use_parent_model_frame>
+        <xyz>0 0 1</xyz>
+        <limit>
+          <lower>-0.401426</lower>
+          <upper>0.5</upper>
+          <effort>0</effort>
+        </limit>
       </axis>
     </joint>
-    
     <joint name='right_hub' type='fixed'>
       <parent>right_tie_rod_arm</parent>
       <child>right_hub</child>
       <pose>0 0 0 0 0 0</pose>
     </joint>
-    
     <joint name='right_pin' type='revolute'>
       <parent>right_tie_rod_arm</parent>
       <child>tie_rod</child>
-      <pose>0 -0.408503 0 0 0 0</pose>
+      <pose>0 -0.817006 0 0 0 0</pose>
```

Comment:
Yes, this is due to Drake's SDF parser internally modifying a link's frame to be equal to its joint's frame: https://github.com/RobotLocomotion/drake/issues/1714#issuecomment-194697369